File: Competitive Programming/DAY-2/star.py
step=int(input("Enter number of staircase required"))
for i in range(1,step+1):
    if i%2==0:
        k=i
    else:
        k=i+1
        
    for g in range(step,k,-1):
        print(" ",end="")
    
    for j in range(1,k+1):
        print("* ",end="")
    print()

File: Competitive Programming/DAY-2/star.cpp
#include <iostream>

using namespace std;


void star(int step)
{
    for(int i=1;i<=step;i++)
    {
        int k=(i%2==0)?i:i+1;
        for(int l=step;l>k;l--)
            cout<<" ";
            
        for(int j=1;j<=k;j++)
            cout<<"* ";
      cout<<endl;  
    }
}
int main()
{
    int n;
    cout<<"Enter number of staircase to be printed";
    cin>>n;
    star(n);
    return 0;
}
